data/data/agent/systemd/units/node-zero.service

```diff
@@ -4,9 +4,10 @@ Wants=network-online.target
 After=network-online.target
 
 [Service]
+Type=oneshot
+RemainAfterExit=True
 ExecStart=/usr/local/bin/set-node-zero.sh
 ExecStartPost=/bin/sleep 5
```

Comment:
I suspect that this sleep existed only to cover up the race condition, and could be removed now.

Reply:
Good point (I also removed it in the initial implementation)